hw/drivers/sensors/bmp388/src/bmp388.c

```diff
@@ -3416,8 +3625,10 @@ bmp388_sensor_read(struct sensor *sensor, sensor_type_t type,
 
     if (cfg->read_mode.mode == BMP388_READ_M_POLL) {
         rc = bmp388_poll_read(sensor, type, data_func, data_arg, timeout);
-    } else {
+    } else if (cfg->read_mode.mode == BMP388_READ_M_STREAM) {
         rc = bmp388_stream_read(sensor, type, data_func, data_arg, timeout);
+    } else { // hybrid mode
```

Comment:
Please convert this C++ style comment to C style comment

Reply:
Done